### PR TITLE
fix: use correct favicons everywhere

### PR DESCRIPTION
**Pull request by alecdwm**
Fixes the wrong favicon showing in the reminder to pin the extension, and also next to the tab title.

### CODE REVIEW

File: apps/extension/src/template.dashboard.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="./favicon.svg" />
+    <link rel="icon" href="./favicon.svg?v=<%= __webpack_hash__  %>" />
```
Comment (alecdwm) [inline apps/extension/src/template.dashboard.html:5]: Browsers aggressively cache favicons, this hash is needed in order to get it to update between dev and canary builds (so when the extension id doesn't change).

File: apps/extension/src/@types/react-app-env.d.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ declare namespace NodeJS {
     readonly SENTRY_AUTH_TOKEN?: string
     readonly SIMPLE_LOCALIZE_API_KEY?: string
     readonly SIMPLE_LOCALIZE_PROJECT_TOKEN?: string
-    readonly BUILD?: "production" | "canary" | "ci"
+    readonly BUILD?: "production" | "canary" | "ci" | "qa" | "dev"
```
Comment (alecdwm) [inline apps/extension/src/@types/react-app-env.d.ts:15]: Doesn't change anything, the type was just missing two possible values

File: apps/extension/webpack/webpack.prod.js [hunk 3]
```diff
@@ -35,10 +39,6 @@ const config = (env) => {
             from: "manifest.json",
             to: distDir,
             context: "public",
-            // overwrite non-transformed manifest.json
```
Comment (alecdwm) [inline apps/extension/webpack/webpack.prod.js:38]: Not needed when we don't copy the manifest.json twice

**Review by 0xKheops — APPROVED** [review]
Cool one, thanks !
LGTM !

**Pull request merged**